packages/ui-components/src/components/windowedlist.ts

```diff
@@ -917,6 +980,13 @@ export class WindowedList<
    * The default implementation of this handler is a no-op.
    */
   protected onUpdateRequest(msg: Message): void {
+    if (this.scrollbar) {
+      // Hide the native scrollbar.
+      this.node.style.width = 'calc(100% + 25px)';
+      this._scrollbar.update();
+    } else {
+      this.node.style.width = 'calc(100%)';
```

Comment:
let's do the math :stuck_out_tongue_winking_eye: 

```suggestion
      this.node.style.width = '100%';
```